### PR TITLE
Revert "Move bazel.rc to workspace root to support bazel-0.18.0"

### PR DESCRIPTION
**Pull request by perfinion**
This reverts commit a74a3217f7ff2dbee2fb618aa658cf666861545c.

Bazel-0.18.0 is changing where it searches for .bazelrc files.
Originally it was removing /tools/bazel.rc and only using /.bazelrc.
This causes issues for gitignoring /.bazelrc and 0.18.0 has temporarily
added tools/bazel.rc back to the list until 0.19. The long term solution
is to use try-import but that statement is new in 0.18 and we are not
going to bump TF's minimum right away. When 0.19 is out things will need
to be changed back and the minimum bumped to 0.18.

Fixes: https://github.com/tensorflow/tensorflow/issues/22762
Fixes: https://github.com/tensorflow/tensorflow/pull/22906
Signed-off-by: Jason Zaman <jason@perfinion.com>

### CODE REVIEW

**Review by gunan — APPROVED** [review]

**Comment by perfinion** [comment]
https://github.com/tensorflow/tensorflow/pull/22964 has the change to use try-import but will need to wait until bazel-0.19 is out.

**Comment by gunan** [comment]
Unassigning, to make the sync rotation pick up the merge when tests are run.

**Review by meteorcloudy — APPROVED** [review]

**Comment by angersson** [comment]
This needed some additional changes to our internal deployment configuration, so I made an internal change that is waiting on approval.

**Comment by drpngx** [comment]
Thanks @angersson !

**Comment by girving** [comment]
Thank you!

**Pull request merged**
